calc.py

```diff
@@ -96,7 +96,8 @@ def safe_input_single_value(prompt_message: str) -> float:
     return value
 
 def safe_input_single_value_int(prompt_message: str) -> int:
-    """Input utility that gets a float value as an output, it is protected using safe_convert"""
+    """Input utility that gets a float value as an output, it is protected using safe_convert.
+   But unlike the one without _int, this returns int instead of float"""

```

Comment:
```suggestion
    """Input utility that gets a float value as an output, it is protected using safe_convert.
    But unlike the one without _int, this returns int instead of float"""
```
This is so it passes the linter.